solver/src/driver.rs

```diff
@@ -258,6 +258,27 @@ impl Driver {
         }
     }
 
+    // Log the amount of orders that were matched but not settled in this runloop (effectively queued for the next one)
+    // Should help us to identify how much we can save by parallelizing execution.
+    fn report_matched_but_unsettled_orders<I: Iterator<Item = Settlement>>(
+        &self,
+        submitted: &Settlement,
+        all: I,
+    ) {
+        let submitted: HashSet<_> = submitted
+            .trades()
+            .iter()
+            .map(|trade| trade.order.order_meta_data.uid)
+            .collect();
+        let all_matched: HashSet<_> = all
+            .flat_map(|solution| solution.trades().to_vec())
+            .map(|trade| trade.order.order_meta_data.uid)
+            .collect();
+        let matched_but_not_settled: HashSet<_> = all_matched.difference(&submitted).collect();
+        self.metrics
+            .orders_matched_but_not_settled(matched_but_not_settled.len())
```

Comment:
The doc string on this method says that we log these values, but the implementation only seems to include it in the metrics.... Do the metrics have a log statement in them?

Reply:
By logging I mean report to a prometheus metrics, which grafana collects from our pods periodically. Will update the documentation to clarify this.